docs/content/FAQ/_index.md

```diff
@@ -59,6 +60,13 @@ ref "/reference/data_directory" >}}).
 
 The Infection Monkey agent shuts off either when it can't find new victims or it has exceeded the quota of victims as defined in the configuration.
 
+## How long does it take to stop all running Infection Monkey agents?
+
+When <b>Kill All Monkeys</b> is pressed, the agents try to finish execution
+safely. This can take up to 2 minutes, but will be much shorter on average.
+
+![Kill All Monkeys Button](/images/faq/kill_all_monkeys.jpeg "Kill All Monkeys Button")
```

Comment:
Do we need this screenshot? I'm not sure what it really adds.

Reply:
I thought just for the users to know about what we are talking about.  Maybe we don't actually need it. Or I can add the modal window screenshot ?